internal/cmd/root.go

```
@@ -35,12 +35,13 @@ func init() {
 	rootCmd.PersistentFlags().BoolVar(&noMultipleTokenSourcesWarning, "no-multiple-token-sources-warning", false, "Don't warn about multiple access token sources")
 
 	rootCmd.PersistentPostRun = func(cmd *cobra.Command, args []string) {
+		settingsV, _ := settings.ReadSettings()
```

Please don't ignore errors